resources/ahk/TradeMacro.ahk

```diff
@@ -87,13 +87,6 @@ OpenSearchOnPoeTrade:
 	SuspendPOEItemScript = 1 ; This allows us to handle the clipboard change event
 	Send ^{sc02E}
 	Sleep 250
-	
-	TradeFunc_DoParseClipboard()
-	If (!Item.Name and TradeOpts.OpenUrlsOnEmptyItem) {
```

Comment:
Why did you remove this? If you don't hover over an item this function is supposed to open the wiki frontpage. 
I also noticed it doesn't work correctly anymore because the old Item.Name is still being used, using this on Facebreakers and then without item hover opens the facebreaker wiki page both times.